src/components/KYCWall/BaseKYCWall.js

```diff
@@ -134,11 +134,7 @@ class KYCWall extends React.Component {
                     shouldShowPaypal={false}
                     onItemSelected={(item) => {
                         this.setState({shouldShowAddPaymentMenu: false});
-                        if (item === CONST.PAYMENT_METHODS.BANK_ACCOUNT) {
-                            Navigation.navigate(this.props.addBankAccountRoute);
-                        } else if (item === CONST.PAYMENT_METHODS.DEBIT_CARD) {
-                            Navigation.navigate(this.props.addDebitCardRoute);
-                        }
+                        PaymentMethods.navigateToAddPaymentMethodPage(item, false);
```

Comment:
Are we sure this will work? It looks like the route depend on the where the KYCWall is being triggered: 

https://github.com/Expensify/App/blob/4649fbc631b2f8e2ff224c47a3e24f51d8621e73/src/ROUTES.js#L61-L62

https://github.com/Expensify/App/blob/4649fbc631b2f8e2ff224c47a3e24f51d8621e73/src/ROUTES.js#L68-L69

Reply:
ah thats a good catch. Thanks